docs/usage/getting-started.md

```diff
@@ -47,235 +47,53 @@ If you just want to run Observatorium locally and get started quickly, take a lo
 
-OpenTelemetry operator can be installed by running the following command:
+- **Jager Operator**: our tool of choice for tracing. See [Jaeger documentation](https://www.jaegertracing.io/docs/latest/operator/) for more information.
 
-```bash
-kubectl apply -f https://github.com/open-telemetry/opentelemetry-operator/releases/latest/download/opentelemetry-operator.yaml
-```
+- **OpenTelemetry Operator**: we use the OTel Collector to build the distributed tracing component. See [open-telemetry/opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator) for more information.
 
-The operator should be installed into `opentelemetry-operator-system` namespace. See [open-telemetry/opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator) for more information.
+- **Token refresher**: the token issued by the OIDC providers often have a small validity, but it can be automatically refreshed using a refresh token. As Prometheus doesn't natively support refresh token flow, we use a proxy in-between to do exactly that for us. Take a look at the [GitHub repo](https://github.com/observatorium/token-refresher) to read more about it.
 
 #### Deploying Observatorium
 
-We will deploy the Observatorium using Kubernetes manifests generated from Jsonnet. If the IP of `docker0` interface was different then the default in the above steps, you will need to update the `tenant.issuerURL` with correct IP address and run `make generate` to recreate the manifests.
-
-Let's deploy `minio` first, as Thanos and Loki have a dependency on it.
-
-```bash
-kubectl create ns observatorium-minio
-kubectl apply -f ./manifests/minio-pvc.yaml
-kubectl apply -f ./manifests/minio-deployment.yaml
-kubectl apply -f ./manifests/minio-service.yaml
-```
-
-Wait for `minio` to come up.
-
-```bash
-kubectl wait --for=condition=available -n observatorium-minio deploy/minio
-```
-
-After `minio` starts running, deploy everything else.
+We will deploy the Observatorium using Kubernetes manifests generated from Jsonnet. To automate everything, we run the quickstart script:
 
-```bash
-kubectl create ns observatorium
-kubectl apply -f ./manifests/
+```sh
+quickstart.sh
 ```
 
-Wait for everything in `observatorium` namespace to come up before moving forward.
+The script will take care of setting up everything.
 
 #### Writing data from outside
 
-The Observatorium API allows you to push timeseries data using Prometheus remote write protocol. The write endpoint is protected, so we need to authenticate as a tenant. The authentication is handled by the OIDC Provider we started earlier. If you take a look at `manifests/api-secret.yaml`, you can see that we have configured a single tenant `test-oidc`. The remote write endpoint is `/api/metrics/v1/<tenant-id>/api/v1/receive`.
-
-#### Token refresher
```

Comment:
This sounds like a useful piece, no? Why we are removing it? 🙈

Reply:
I moved all the information about the different components being installed to a simple list under the pre-requisites header. The diff doesn't help making this easy to see, but i.e. this description is here: https://github.com/observatorium/observatorium/pull/479/files#diff-a22e1573bc4b33b17de5b7f0e2017009c4308d2848d35fbdf1cf4f6dbadd5556R71

The idea is to give a small summary there and link to each project's own page/README.